src/pages/waiting-room-to-car/cat-b/__tests__/waiting-room-to-car.cat-b.page.spec.ts

```diff
@@ -37,6 +32,11 @@ import { TellMeQuestionSelected, EyesightTestReset } from '../../../../modules/t
 import { PracticeModeBanner } from '../../../../components/common/practice-mode-banner/practice-mode-banner';
 import { WaitingRoomToCarValidationError } from '../../waiting-room-to-car.actions';
 import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { TellMeQuestionCardComponent } from '../components/tell-me-question-card/tell-me-question-card';
+import { TellMeQuestionComponent } from '../components/tell-me-question/tell-me-question';
+import { TellMeQuestionOutcomeComponent } from '../components/tell-me-question-outcome/tell-me-question-outcome';
+import { TransmissionComponent } from '../components/transmission/transmission';
+import { InstructorRegistrationComponent } from '../components/instructor-registration/instructor-registration';
```

Comment:
We should be mocking these in this test file to make sure we are only testing the functionality that has been added by this page. MockComponents() is what we usually use